### PR TITLE
chore: only use new calico CSE config for AKS Engine

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

AKS will have to integrate changes from #454 distinctly. This PR will enable AKS to do so.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jackfrancis** [comment]
@feiskyer FYI

File: parts/k8s/kubernetesconfigs.sh [hunk 1]
```diff
@@ -198,13 +211,20 @@ configureCNI() {
...
+    if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
+        if [[ "${NETWORK_POLICY}" != "calico" ]]; then
+            mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
+            chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
+        fi
+        /sbin/ebtables -t nat --list
     fi
 }
 
+if [ "$IS_HOSTED_MASTER" = "false" ]; then
```
Comment (CecileRobertMichon) [inline parts/k8s/kubernetesconfigs.sh]: what is this doing? there's no body

Comment (jackfrancis) [reply parts/k8s/kubernetesconfigs.sh]: Just a test to see if you were paying attention (you passed)

**Comment by jackfrancis** [comment]
Also validated calico + AKS Engine

File: parts/k8s/kubernetesconfigs.sh [hunk 2]
```diff
@@ -190,6 +190,19 @@ configureCNI() {
     # needed for the iptables rules to work on bridges
     retrycmd_if_failure 120 5 25 modprobe br_netfilter || exit $ERR_MODPROBE_FAIL
     echo -n "br_netfilter" > /etc/modules-load.d/br_netfilter.conf
+    if [ "$IS_HOSTED_MASTER" = "true" ]; then
```
Comment (mboersma) [inline parts/k8s/kubernetesconfigs.sh]: Should this be squiggly-bracketed like the other env vars? `"${IS_HOSTED_MASTER}"`

File: parts/k8s/kubernetesconfigs.sh [hunk 3]
```diff
@@ -198,10 +211,15 @@ configureCNI() {
...
-    if [[ "${NETWORK_PLUGIN}" = "cilium" ]]; then
-        systemctl enable sys-fs-bpf.mount
-        systemctl restart sys-fs-bpf.mount
-        REBOOTREQUIRED=true
+}
+
+configureCNIIPTablesAKS() {
+    if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
+        if [[ "${NETWORK_POLICY}" != "calico" ]]; then
+            mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
```
Comment (mboersma) [inline parts/k8s/kubernetesconfigs.sh]: Maybe quote and squiggle-bracket `$CNI_BIN_DIR`? (We should bite the bullet and `shellcheck` all these at some point.)

**Pull request merged**